etc/parameters/default.json

```diff
@@ -1111,7 +1111,10 @@
       "striker_supporter_minimum_x": 2.0,
       "keeper_x_offset": 0.1,
       "striker_distance_to_non_free_center_circle": 0.4,
-      "striker_set_position": [-0.3, 0.0]
+      "striker_kickoff_pose": {
+        "translation": [0.0, 0.3],
+        "rotation": [0.0007962742820382118, -0.9999997019767761]
```

Comment:
Do you intend to have it turned by 90 degrees? Then better put a clear `0.0` and `-1.0`

Reply:
Good point, I serialized it to see what the proper format was and then didn't think about rounding errors. Fixed now